easybuild/easyconfigs/f/Flang/Flang-20190329-GCCcore-8.2.0-CUDA-10.1.105.eb

```diff
@@ -0,0 +1,69 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2013-2015 Dmitri Gribenko, Ward Poelmans
+# Authors:: Dmitri Gribenko <gribozavr@gmail.com>
+# Authors:: Ward Poelmans <wpoely86@gmail.com>
+# License:: GPLv2 or later, MIT, three-clause BSD.
+# $Id$
+##
+
+name = 'Flang'
+version = '20190329'
+llvm_version = '7.1.0'
+cudaver = '10.1.105'
+versionsuffix = '-CUDA-%s' % (cudaver)
+
+homepage = 'https://github.com/flang-compiler/'
+description = """Fortran compiler, based on LLVM."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+# Do not set optarch to True: it will cause the build to fail
+toolchainopts = {'optarch': False}
+
+sources = [
+    {
+        'download_filename': 'flang_%(version)s.tar.gz',
```

Comment:
Yes, all the files have the same name in the different repos...what a great idea!

Reply:
🤦‍♂